### ISSUE TITLE
Make unlimited max_stack_size as default value

### ISSUE THREAD

**Issue by Nordsoft91** [peter-kish/gloot#206]
Make max_stack_size unlimited if not defined in protoset for StackedInventory

**Comment by LeeWannacott** [comment]
I think max_stack_size should be one by default, I guess it depends on the genre of game...

**Comment by peter-kish** [comment]
Yeah, it depends on the particular use case, though I still think that 100 (the default in v2.4.10) is not really useful.

This is changed in v3.0.0, where all items are considered item stacks where the default maximum stack size is one.

**Comment by peter-kish** [comment]
Version 3.0.0 has been released and the default stack_size and max_stack_size is 1.
